packages/cuba-rest-js/src/cuba.ts

```diff
@@ -334,13 +334,15 @@ export class CubaApp {
   }
 
   public getPermissions(fetchOptions?: FetchOptions): Promise<PermissionInfo[]> {
-    return this.fetch('GET', 'v2/permissions', null, {handleAs: 'json', ...fetchOptions});
+    // https://github.com/cuba-platform/frontend/issues/177
+    // security model is changed in cuba rest 7.2 so we do not support roles and permission at this moment
+    return Promise.reject(CubaApp.NOT_SUPPORTED_BY_API_VERSION);
```

Comment:
I know it looks like a correct fix, but consider this:
no matter if we patch react-core@0.1.0 or not, if `rest` will be updated alone, it will break things. I suggest to return empty array (ugly but won't break things)